### PR TITLE
Fix OS X build

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
@jafyvilla FYI

**Comment by jafyvilla** [comment]
+1

File: livre/core/util/FrameUtils.cpp [hunk 1]
```diff
@@ -58,7 +58,7 @@ uint32_t FrameUtils::getNext( uint32_t current, const int32_t delta ) const
     if( _frameRange == INVALID_FRAME_RANGE )
         return INVALID_FRAME;
 
-    const uint32_t interval = std::abs( _frameRange[1] - _frameRange[0] );
+    const uint32_t interval = _frameRange[1] - _frameRange[0];
```
Comment (bilgili) [inline livre/core/util/FrameUtils.cpp]: - Why did you remove the abs ? what if animation is on the other direction ?

Comment (jafyvilla) [inline livre/core/util/FrameUtils.cpp]: (_frameRange[1] - _frameRange[0]) can never give a negative result, as frameRange is unsigned. The animation can still go on the other direction and this would still work, afaik

Comment (eile) [inline livre/core/util/FrameUtils.cpp]: abs( unsigned ) is a nop, and clang complains.

Comment (rdumusc) [inline livre/core/util/FrameUtils.cpp]: I don't see anywhere a check that frameRange[1] >= frameRange[0] though. This must be enforced and unit tested!
